### DISCUSSION TITLE
Micro SD Card Lock after using Onion Desktop Tool

### DISCUSSION THREAD

**Discussion by VhoTheReaper** [OnionUI/Onion#1303] [General]
Hello,

I use Onion Desktop Tools on new SD Card for my Miyoo Mini Plus, but I want use this for other device but it's impossible to format, it seems lock, have you an idee how I can format my card ?

Thank you

**Comment by schmurtzm** [comment]
As alternative you can use Rufus : 

https://onionui.github.io/docs/installation/fresh

**Reply by VhoTheReaper** [reply]
This doesn't work with rufus either

**Comment by 0023-119** [comment]
You probably need to use some proper partitioning tool and repartition your card from the scratch. Assuming you don't have the lock turned on on the SD adapter or something

**Reply by VhoTheReaper** [reply]
There is no physical switch on the micro SD card, I tried with GParted, same, nothing works, the card still keeps the onionOS data

**Reply by schmurtzm** [reply]
Do you have another SD card reader ?

**Reply by VhoTheReaper** [reply]
Yes I haven't tried it yet
